### PR TITLE
Fix HTML Overload Issue

### PR DESCRIPTION
**Pull request by MrDogeBro**
Fixes the issue where the project was overloaded with html from images being compressed in, therefore making GitHub show it as an HTML project when it is actually a Python project.

### Changes
- Move all assets into assets folder at top level
   - `logo.png` is now directly in there, the svg icons are in the `icons` folder
- Update ignore to include pngs and svgs from the assets folder
- Update html to link to GitHub repo for icons and update logo link to new location

Hopefully this should be all good. If not, just let me know and I can get it changed. Thanks!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -224,3 +224,7 @@ dmypy.json
 
 # profiling data
 .prof
+
+# include assets
+!assets/**/*.png
+!assets/**/*.svg
```
Comment (parafoxia) [inline .gitignore:230]: These should be at the top under the Custom section.

**Review by parafoxia — CHANGES_REQUESTED** [review]
Just a small change, but everything else looks good.

**Comment by parafoxia** [comment]
Just a note: some extra commits have been pushed to main; you should **not** need to merge and rebase those changes.

**Comment by parafoxia** [comment]
I've also just realised, you'll need to update the filepath in `docs/conf.py` -- changing it to `../assets/logo.png` should work, but maybe worth building the docs and checking just to absolutely make sure.

File: .gitignore (hunk 1 above)
Comment (MrDogeBro) [reply .gitignore]: Yup sounds good. That's now done.

**Comment by MrDogeBro** [comment]
Ok that small change should now be good. I also updated the logo file path in the docs config and it still works fine when I built it locally so everything should be good on that front (I see no reason why not). So I think everything should now be good then.

**Review by parafoxia — APPROVED** [review]
Yep, everything looks good as far as I can see.

**Pull request merged**